R-package/configure.ac

```diff
@@ -0,0 +1,155 @@
+### configure.ac                    -*- Autoconf -*-
+# Template used by Autoconf to generate 'configure' script. For more see:
+#   * https://unconj.ca/blog/an-autoconf-primer-for-r-package-authors.html
+#   * https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Configure-and-cleanup
+
+AC_PREREQ(2.69)
+AC_INIT([lightgbm], [2.3.2], [], [lightgbm], [])
```

Comment:
Read version from file: https://github.com/microsoft/LightGBM/pull/3188#discussion_r449729982.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/3188/commits/d78932e72f000500bbe0e7c4761b6c29c867fd54. Good suggestion!